src/components/cards/card-booking/index.css

```
@@ -0,0 +1,39 @@
+.container-button-booking {
+  margin: 0px 16px 16px 16px;
+}
+
+.button-booking {
+  margin-top: 16px;
+}
+
+.container-user {
+  display: flex;
+  gap: 8px;
+  margin: 16px 16px 16px 16px;

```

```suggestion
  margin: 16px;
```

Depois dá uma olhada sobre essa property do css: https://www.w3schools.com/css/css_margin.asp, mas vc pode ter alguns atalhos. Essa forma de escrever, por exemplo, aplica aquela medida nos 4 lados.